### DISCUSSION TITLE
Any plan to for technical writers and creators?

### DISCUSSION THREAD

**Discussion by MarcelCloud** [NillionNetwork/discussions#55] [Ideas]
I believe a project like Nillion should have participation plans in place for technical writers and creators. It could come in the form of documentation review and feedback, and tech video contests (reviews reviewing the [https://docs.nillion.com/guide-testnet-connect](url) etc.

These are the meta currently championing community growth in the Web3 Eco-system even for Devs.

**Comment by Comfy242** [comment]
I would as well buy your idea.  There should be a way technical writers and content creators can come in

**Comment by oceans404** [comment]
Great idea! We could test this idea by creating a bounty around technical writing / content. Are there specific types of content or topics that you would be interested in creating content on Nillion around?

**Comment by MarcelCloud** [comment]
@oceans404 thank you for responding to my recommendation. Here's what I think:

— a set of bounty tasks is perfect. Who doesn't love incentives for doing what they love?

— **_the nillion introductory video that explains the Nillion blockchain as it relates to decentralization and on-chain security (Blind Computation) is a good starting point. Participants are expected to create an extensive Twitter thread from that video touching all major keywords._**

—I have participated in a series of thread bounties and articles at superteamEarn — so Twitter thread bounties, articles, guides, and video walkthroughs talking about Nillion, her value prop, and how to participate in the Testnet.

Potential topical focus includes but is not limited to;
_1. Blind Computation — Explained
2. What makes Nillion different from the existing cryptographic primitives?
3. And of course, if they have APIs or any dev tools for devs to build on__

Open to further review and suggestions.
